adcircpy/fort15.py

```diff
@@ -2271,6 +2271,8 @@ def _get_TOUTS__(self, output_type, physical_var):
                 start = start - self.forcing_start_date
             else:
                 start = start - self.start_date
+        elif isinstance(start, datetime):
+            start = self.start_date - start
```

Comment:
in case `.start_time` is set as a `datetime` object, this derives the appropriate timedelta

Reply:
It *always* is a datetime object.